### ISSUE TITLE
Locale fallback

### ISSUE THREAD

**Issue by preusx** [QingWei-Li/docsify#152]
I've just installed the docsify-cli. During the execution of the `init` command:
```bash
docsify init ./docs
```
I get the following error:
```bash
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/usr/lib/node_modules/docsify-cli/tools/locales/ru.json'
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:491:33)
    at Object.readFileSync (/usr/lib/node_modules/docsify-cli/node_modules/jsonfile/index.js:66:20)
    at Locales._existsLocaleFile (/usr/lib/node_modules/docsify-cli/tools/locales/index.js:32:22)
    at Locales.detectLocale (/usr/lib/node_modules/docsify-cli/tools/locales/index.js:22:25)
    at new Locales (/usr/lib/node_modules/docsify-cli/tools/locales/index.js:13:20)
    at Object.<anonymous> (/usr/lib/node_modules/docsify-cli/bin/docsify:13:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

Apparently, there is no locale file for Russian(and not only). Any other command is causing the same error.

It can be "fixed", for example, by calling the cli like that(linux at least):
```bash
LANG=en_US.UTF-8 docsify 
```
But fallback to english locale could be more applicable.
Thank you.

**Comment by QingWei-Li** [comment]
@jrappen Can you fix it please? 😄

**Comment by jrappen** [comment]
Related: QingWei-Li/docsify-cli@02dec13 (see comments there at the bottom of the page)

Currently I don't know how to fix this. Two possible temporary fixes:

* remove i18n stuff
* add Russian translation (@preusx)

**Comment by preusx** [comment]
Well, error happens when `fs-extra` module tries to parse file that doesn't exist. Although there is an option `throws: false` have been passed. But as described in [documetation](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md) throws will only catch JSON parsing error, but not IO.

The simplest solution is to wrap [this line](https://github.com/QingWei-Li/docsify-cli/blob/master/tools/locales/index.js#L32) with `try { .. } catch(e) { return false }` expression. Or check path esistance with [`fs.existsSync()`](https://nodejs.org/api/fs.html#fs_fs_existssync_path)

**Comment by QingWei-Li** [comment]
fixed in `docsify-cli@3.3.2`
